pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -1,82 +1,37 @@
-{ stdenv, lib, fetchFromGitHub
-, cmake, pkg-config, qttools
-, libxcb, libXau, pam, qtbase, wrapQtAppsHook, qtdeclarative
-, qtquickcontrols2 ? null, systemd, xkeyboardconfig
+{
+  lib,
+  callPackage,
+  runCommand,
+  qtwayland,
+  wrapQtAppsHook,
+  unwrapped ? callPackage ./unwrapped.nix {},
+  withWayland ? false,
```

Comment:
What speaks against activating wayland all the time like for many other pakcages?

Reply:
It's considered experimental upstream, and it needs some extra config file tweaks to actually be enabled (see the module for that).

Reply:
Does this mean we can drop https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/display-managers/sddm.nix#L227 ?

Reply:
_Technically_ SDDM can run pure Wayland now, but I'm not sure what else that option does that might be relevant.

Reply:
Then I am going adventurous again and going to trythat. Before I just got my terminal but that was before the other fixes done by you.

Reply:
Can someone please be adventurous enough to rewrite the services.xserver module...

Reply:
👀  sneakpeek https://c3d2.social/@sandro/112000600811040803

see you in the next PR

Reply:
First PR #291913